.github/workflows/approve-and-merge-dispatch.yml

```
@@ -0,0 +1,48 @@
+name: Approve and Merge Command Dispatch
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  approveAndMergeDispatch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Auto Approve Slash Command Dispatch
+        uses: peter-evans/slash-command-dispatch@v3
+        id: scd
+        with:
+          token: ${{ secrets.GH_PAT_APPROVINGTON_OCTAVIA }}
+          permission: admin
+          issue-type: pull-request
+          repository: airbytehq/airbyte-cloud
```

```suggestion
          repository: airbytehq/airbyte-platform-internal
```
This was using the old repo name; I'm guessing it still works since github redirects the old name to the new one, but we may as well be consistent.

---

Yup! Will change